src/session.rs

```diff
@@ -75,6 +75,7 @@ impl<FS: Filesystem> Session<FS> {
             && !(options.contains(&MountOption::AllowRoot)
                 || options.contains(&MountOption::AllowOther))
         {
+            warn!("Given auto_unmount without allow_root or allow_other; adding allow_other");
```

Comment:
can you add `adding allow_other, with userspace permission handling`, or something to that effect?

Reply:
Fixed!